app/views/layouts/application.html.erb

```diff
@@ -13,9 +13,15 @@
       <%= link_to "Home Page", "/"%>
       <%= link_to "All Merchants", "/merchants"%>
       <%= link_to "All Items", "/items"%>
-      <%= link_to "Cart: #{cart.total_items}", "/cart" %>
-      <%= link_to "Login", "/login"%>
-      <%= link_to "Register", "/register"%>
+      <%= link_to "Cart: #{cart.total_items}", "/cart" unless current_admin? %>
+      <%= link_to "Login", "/login" unless current_user%>
+      <%= link_to "Register", "/register" unless current_user%>
+      <%= link_to "Profile", "/profile"%>
+      <%= link_to "Logout", "/logout"%>
```

Comment:
`<%= link_to "Logout", "/logout", method: :delete %>` so that it matches the route @danielpselph proposed in Slack